recipes/esmp/build.sh

```diff
@@ -0,0 +1,105 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC ${CFLAGS}"
+export CXXLAGS="${CFLAGS} ${CXXLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+export ESMF_DIR=`pwd`"/esmf"
+export ESMP_source="ESMP"
+export ESMF_PTHREADS="OFF"
+export ESMF_OS=`uname -s`
+
+export ESMF_COMPILER="gfortran"
+export ESMF_ABI="64"
+if [ `uname` == Darwin ]; then
+    export ESMF_OPENMP="OFF"
+    export CC="/usr/bin/gcc"
+    export FC=${PREFIX}/bin/gfortran
+    export F77=${PREFIX}/bin/gfortran
+    export F90=${PREFIX}/bin/gfortran
+else
+    export ESMF_OPENMP="ON"
+fi
+
+# OPENMPI bits
+CONDA_LST=`conda list`
+if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
```

Comment:
The right way to do this would be with features. Right now we cannot ensure that this will work expect at build time. I recommend you to choose one (`openmpi`?) and add the other later once we have implemented features.